CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 All notable changes to this project will be documented in this file. This
 project adheres to [Semantic Versioning](http://semver.org/).
 
+## [0.12.0] - 2019-09-24
```

Comment:
Do you need to add this link to the footer of this file (you can guess what the URL will be based on the tag)?